scripts/prepare/bundle.ts

```diff
@@ -32,40 +32,40 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
     await fs.emptyDir(join(process.cwd(), 'dist'));
   }
 
-  if (!optimized) {
-    await Promise.all(
-      entries.map(async (file: string) => {
-        console.log(`skipping generating types for ${file}`);
-        const { name: entryName, dir } = path.parse(file);
-
-        const pathName = join(process.cwd(), dir.replace('./src', 'dist'), `${entryName}.d.ts`);
-        const srcName = join(process.cwd(), file);
-
-        const rel = relative(dirname(pathName), dirname(srcName))
-          .split(path.sep)
-          .join(path.posix.sep);
-
-        await fs.ensureFile(pathName);
-        await fs.writeFile(
-          pathName,
-          dedent`
-          // devmode
-          export * from '${rel}/${entryName}';
-        `
-        );
-      })
-    );
-  }
```

Comment:
Moving this should get a little speed increase, since we're not waiting for this to complete before starting the esbuild builds.